transforms/no-implicit-this/__testfixtures__/-mock-telemetry.json

```diff
@@ -5,58 +5,6 @@
   "block-component": { "type": "Component" },
   "foo": { "type": "Component" },
   "namespace/foo": { "type": "Component" },
-  "built-in-helpers": {
```

Comment:
many of these were unneeded. :bowing_man:

Reply:
they were needed because the usage of `getTelemetryFor()` needed them to exist